src/main/scala/pureconfig/package.scala

```diff
@@ -76,6 +76,20 @@ package object pureconfig {
   }
 
   /**
+    * Load a configuration of type `Config` from the given `Config`, falling back to the default configuration */
```

Comment:
Can I ask you to format the doc as for the other methods and to add a description for the parameters (copy-pasted from the other methods).